scrapers/VegasDreamsLLC.yml

```diff
@@ -34,5 +39,26 @@ xPathScrapers:
                https://helloladyboy.com: HelloLadyBoy
                https://milftrip.com: MilfTrip
                https://screwmetoo.com: ScrewMeToo
+  performerScraper:
+    performer:
+      Name: //*/h1[@class="amp-model-title"]/text()
+      Measurements: //*/span[@class="model-details" and text()="Measurements:"]/../text()
+      Height:
+        selector: //*/span[@class="model-details" and text()="Height:"]/../text()
+        postProcess:
+          - feetToCm: true
+      Weight:
+        selector: //*/span[@class="model-details" and text()="Weight:"]/../text()
+        postProcess:
+          - replace:
+              - regex: .*([0-9]{2,3}).*
+                with: $1
```

Comment:
regex doesn't work <https://regex101.com/r/9Y7Emp/1>
why not use something simpler like 
```yaml
              - regex: ([^\d]+)
                with: ""
```
to clean the weight?